bestla/bestla/kernel_ref.h

```diff
@@ -181,7 +181,14 @@ static inline BTLA_CODE compress_f4(const int8_t* srcptr, utils::f4x2* dstptr, i
 static inline BTLA_CODE compress_3bit(const int8_t* srcptr, bestla::utils::bit2x4* bit2ptr, utils::bit1x8* bit1ptr,
                                       int row, int col, int ld_src, int ld_dst) {
   assert(col % 128 == 0);
-
+  auto round3bit = [](int8_t src) {
+    int32_t dst = src;
+    dst = dst >= 0 ? dst + 16 : dst - 16;
```

Comment:
why ```+16```, ```-16```?

Reply:
I guess it is half of 32 (i.e. `1 >> (8bit - 3bit)`) and it is here to show the behavior of _round to nearest_. 

However, I warried that if it will cause overflow?

Reply:
yes, half of 32. There is a clip after these lines.

Reply:
I see. `dst` is `int32_t` instead of `int8_t`.